### DISCUSSION TITLE
Static MJPEG Streams & Dynamic MJPEG Streams :: Port 8888?

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#200] [Q&A] [answered]
**Quick Question:** Are Static MJPEG Streams & Dynamic MJPEG Streams always going to be **port 8888**? If so, is it port 8888 on the local host or on the container? (Asking because I don't see anything "LISTENING" on that port in either location.)

**Comment by roflcoopter** [comment, marked as answer]
Yes they are exposed on port 8888 inside to container. To access it you need to properly map it when you start your container by adding `-p 8888:8888` to your run command

**Reply by gniknalu** [reply]
> To access it you need to properly map it when you start your container by adding `-p 8888:8888` to your run command

That is the piece I was missing. Thank you!
